web-client/public/js/graph.js

```diff
@@ -1,4 +1,5 @@
 import Grid from "d3-v4-grid";
+import { grnState } from "./grnstate";
```

Comment:
@dondi additionally, this doesn't seem to allow me to call grnState.normalizationMax from grnstate